packages/bitcore-node/test/integration/models/block.integration.ts

```diff
@@ -0,0 +1,611 @@
+        processed: true
+      });
+      await BlockModel.create({
+        chain: 'BTC',
+        network: 'regtest',
+        height: 7,
+        hash: '3279069d22ce5af68ef38332d5b40e79e1964b154d466e7fa233015a34c27312',
+        version: '536870912',
+        merkleRoot: '8c29860888b915715878b21ce14707a17b43f6c51dfb62a1e736e35bc5d8093f',
+        time: 1526326785,
+        nonce: '3',
+        previousBlockHash: '2a883ff89c7d6e9302bb4a4634cd580319a4fd59d69e979b344972b0ba042b86',
+        size: 264,
+        bits: parseInt('207fffff', 16).toString(),
+        processed: true,
+      });
+
+      await TransactionModel.create({
+        txid: 'a2262b524615b6d2f409784ceff898fd46bdde6a584269788c41f26ac4b4919e',
+        chain: 'BTC',
+        network: 'regtest',
+        blockHash: '528f01c17829622ed6a4af51b3b3f6c062f304fa60e66499c9cbb8622c8407f7',
+        blockTime: 1526326784,
+        coinbase: true,
+        locktime: 0,
+        size: 145,
+        blockHeight: 8
+      });
+      await TransactionModel.create({
+        txid: '8a351fa9fc3fcd38066b4bf61a8b5f71f08aa224d7a86165557e6da7ee13a826',
+        chain: 'BTC',
+        network: 'regtest',
+        blockHash: '2a883ff89c7d6e9302bb4a4634cd580319a4fd59d69e979b344972b0ba042b86',
+        blockTime: 1526326785,
+        coinbase: true,
+        locktime: 0,
+        size: 145,
+        blockHeight: 8
+      });
+      await TransactionModel.create({
+        txid: '8c29860888b915715878b21ce14707a17b43f6c51dfb62a1e736e35bc5d8093f',
+        chain: 'BTC',
+        network: 'regtest',
+        blockHash: '3279069d22ce5af68ef38332d5b40e79e1964b154d466e7fa233015a34c27312',
+        blockTime: 1526326785,
+        coinbase: true,
+        locktime: 0,
+        size: 145,
+        blockHeight: 3
+      });
+
+      await CoinModel.create({
+        network: 'regtest',
+        chain: 'BTC',
+        mintTxid: 'a2262b524615b6d2f409784ceff898fd46bdde6a584269788c41f26ac4b4919e',
+        mintIndex: 0,
+        mintHeight: 8,
+        coinbase: true,
+        value: 500.0,
+        address: 'mkjB6LmjiNfJWgH4aP4v1GkFjRcQTfDSfj'
+      });
+      await CoinModel.create({
+        network: 'regtest',
+        chain: 'BTC',
+        mintTxid: '8a351fa9fc3fcd38066b4bf61a8b5f71f08aa224d7a86165557e6da7ee13a826',
+        mintIndex: 0,
+        mintHeight: 9,
+        coinbase: true,
+        value: 500.0,
+        address: 'mkjB6LmjiNfJWgH4aP4v1GkFjRcQTfDSfj'
+      });
+      await CoinModel.create({
+        network: 'regtest',
+        chain: 'BTC',
+        mintTxid: '8c29860888b915715878b21ce14707a17b43f6c51dfb62a1e736e35bc5d8093f',
+        mintIndex: 0,
+        mintHeight: 5,
+        coinbase: true,
+        value: 500.0,
+        address: 'mkjB6LmjiNfJWgH4aP4v1GkFjRcQTfDSfj',
+        spentTxid: 'eec8570a0c960b19fa6c86c71a06ebda379b86b5fe0be0e64ba83b2e0a3d05a3',
+        spentHeight: 9
+      });
+
+      const localTip = await BlockModel.getLocalTip({ chain: 'BTC', network: 'regtest' });
```

Comment:
remove the expects before `handleReorg is called` just tests `handleReorg`

Reply:
resolved

Reply:
confirmed